File: src/main/java/com/p2p/lending/dao/WithdrawalDao.java
package com.p2p.lending.dao;



import java.util.List;
import java.util.Map;

import com.p2p.lending.entity.Withdrawal;


public interface WithdrawalDao {
	List<Withdrawal> withdrawallist(Map<String, Object> map);
	int withdrawalcount(Map<String, Object> ma);
	List<Withdrawal> selectallw();
	Withdrawal selectone(int id);
	int updwith(Map<String, Object> ma);
	int updwiths(Map<String, Object> ma);
	int updmoney(Map<String, Object> ma);
	int intmoney(Map<String, Object> ma);
	int sumtxmoney();
	int sumdzmoney();
	int sumsxf();
}

File: src/main/java/com/p2p/lending/dao/WithdrawalDao.kt
package com.p2p.lending.dao


import com.p2p.lending.entity.Withdrawal


interface WithdrawalDao {

    fun withdrawallist(map: Map<String, Any>): List<Withdrawal>

    fun withdrawalcount(ma: Map<String, Any>): Int

    fun selectallw(): List<Withdrawal>

    fun selectone(id: Int): Withdrawal

    fun updwith(ma: Map<String, Any>): Int

    fun updwiths(ma: Map<String, Any>): Int

    fun updmoney(ma: Map<String, Any>): Int

    fun intmoney(ma: Map<String, Any>): Int

    fun sumtxmoney(): Int

    fun sumdzmoney(): Int

    fun sumsxf(): Int

}
